lib/services/messenger/translators/flowdock.ts

```diff
@@ -62,12 +62,14 @@ export class FlowdockTranslator extends TranslatorScaffold implements Translator
 	): TranslatorMetadata {
 		const superFormat = (format === MetadataEncoding.Flowdock) ? MetadataEncoding.HiddenMD : format;
 		const metadata = TranslatorScaffold.extractMetadata(message, superFormat, config);
-		const findHashtag = new RegExp(`(?:^|\\W)#${config.publicity.shown}(?:\\W|$)`, 'i');
+		const findPublic = '^> ';
```

Comment:
My only worry with this is that `>Hello here's a response` is going to get missed. Does there need to be a space in it?